public/js/login.js

```diff
@@ -0,0 +1,32 @@
+const loginFormHandler = async (event) => {
+  event.preventDefault();
+
+  // trim = leerzeichen vorne und hinten abschneiden
+  const username = document.querySelector("#username").value.trim();
+  const password = document.querySelector("#password").value.trim();
+
+  if (username && password) {
+    const response = await fetch("/api/user/login", {
+      method: "POST",
+      body: JSON.stringify({ username, password }),
+      headers: { "Content-Type": "application/json" },
+    });
+
+    if (response.ok) {
+      // Wenn alles okay, dann geht er direkt zum Dashboard
+      document.location.replace("/dashboard");
+      const dashboardNode = document.querySelector("#dashboard");
+      console.log(dashboardNode);
+      dashboardNode.classList.remove("disabled");
+      const logoutNode = document.querySelector("#logout");
+      logoutNode.classList.remove("disabled");
```

Comment:
Consider creating a partial for a private navbar that you can display depending on whether the user is loggedIn or not in place of the logic here as it will help to make your app more scalable.